### PR TITLE
add research notes file

### LINKED ISSUE #42: Create a md file to track research
Sometimes we'll have tasks where want to research certain things, like new technology we might want to add in the app. 

Create a file that we can organize any notes from these research separately from actually taking actions on the research. 

Spend a few minutes doing a little research to see if anyone else is doing something like this and maybe get some ideas for how to structure it, but if there are no other ideas, simply create a file called `research-notes.md` and add it to the `docs` folder.

### PR DESCRIPTION
**Pull request by Mannuel25**
#### Describe your changes
I added a research notes markdown file to keep track of all researches made.
#### Link to issue this resolves
Fixes: #42 

- [x] The **title** should be something descriptive about what you're changes do. _(It will default to whatever you put as your commit message.)_
- [x] This **description** should include ...
  - [x] A short summary of your changes
  - [x] A link to the issue that relates to these changes
  - [ ] A screenshot of the changes if possible!
- [x] Add `brittanyjoiner15` as a reviewer
- [x] Add yourself as assignee

### CODE REVIEW

**Comment by Mannuel25** [comment]
I made researches as regards a format that'd fit in for such a file like this. Unfortunately, I couldn't find one, so i thought of a format and constructed the markdown file. 

Kindly let me know if the format is okay, or requires necessary corrections. Thanks.

File: docs/research-notes.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#### This file includes research done to assist improve the app, including suggestions for new features or technologies.
```
Comment (brittanyjoiner15) [inline docs/research-notes.md]:
```suggestion
#### This file includes research done to improve the app, including suggestions for new features or technologies.
```

**Review by brittanyjoiner15 — CHANGES_REQUESTED** [review]
Just a quick nitpick to change the wording, but i think this does work! If you can fix that in the text, I think this is good to go!

File: docs/research-notes.md (hunk 1 above)
Comment (Mannuel25) [reply docs/research-notes.md]: Fixed.

**Review by brittanyjoiner15 — APPROVED** [review]
Looks good! I'll merge this on the 1st!

**Comment by Mannuel25** [comment]
> Looks good! I'll merge this on the 1st!

Thank you!

**Comment by brittanyjoiner15** [comment]
@Mannuel25 merging this now!

**Comment by Mannuel25** [comment]
> @Mannuel25 merging this now!

Thank you 😊

**Pull request merged**